src/components/RefineBar/PerCapita.jsx

```diff
@@ -1,4 +1,3 @@
-import './PerCapita.less'
```

Comment:
This would be a good candidate to also convert to a functional component like the narratives button

Reply:
I'll get this in another PR

Reply:
https://github.com/cfpb/ccdb5-ui/pull/398